crates/db_views_actor/src/comment_reply_view.rs

```diff
@@ -276,12 +265,12 @@ impl<'a> CommentReplyQuery<'a> {
       query = query.filter(person::bot_account.eq(false));
     };
 
-    query = match self.sort.unwrap_or(CommentSortType::Hot) {
+    query = match self.sort.unwrap_or(CommentSortType::New) {
       CommentSortType::Hot => query
         .then_order_by(hot_rank(comment_aggregates::score, comment_aggregates::published).desc())
```

Comment:
When I redo these to create lists of multiple types, I'll probably only allow the `New` (published desc) sort anyway.

Reply:
Comment replies will be part of the notification types, which really should only need the new sort anyway.

Reply:
Ah I thought this was the normal CommentView but its a different one.